plugin/src/main/java/eu/maveniverse/maven/njord/plugin3/StatusMojo.java

```diff
@@ -0,0 +1,147 @@
+                        cp.getDistributionManagement().getRepository().getId(),
+                        "default",
+                        cp.getDistributionManagement().getRepository().getUrl())
+                .setSnapshotPolicy(new RepositoryPolicy(false, null, null))
+                .build();
+        RemoteRepository deploymentSnapshot = new RemoteRepository.Builder(
+                        cp.getDistributionManagement().getSnapshotRepository().getId(),
+                        "default",
+                        cp.getDistributionManagement().getSnapshotRepository().getUrl())
+                .setReleasePolicy(new RepositoryPolicy(false, null, null))
+                .build();
+        String deploymentReleaseUrl =
+                artifactDeployerRedirector.getRepositoryUrl(ns, deploymentRelease, RepositoryMode.RELEASE);
+        String deploymentSnapshotUrl =
+                artifactDeployerRedirector.getRepositoryUrl(ns, deploymentSnapshot, RepositoryMode.SNAPSHOT);
+
+        logger.info("Project deployment:");
+        logger.info("* RELEASE");
+        logger.info("  ID: {}", deploymentRelease.getId());
+        logger.info("  POM URL: {}", deploymentRelease.getUrl());
+        if (!Objects.equals(deploymentRelease.getUrl(), deploymentReleaseUrl)) {
+            logger.info("  Effective URL: {}", deploymentReleaseUrl);
+            if (deploymentReleaseUrl.startsWith("njord:")) {
+                ArtifactStoreTemplate template =
+                        ns.selectSessionArtifactStoreTemplate(deploymentReleaseUrl.substring("njord:".length()));
+                logger.info("  Template:");
+                printTemplate(template, false);
+            }
+        }
+        logger.info("* SNAPSHOT");
+        logger.info("  ID: {}", deploymentSnapshot.getId());
+        logger.info("  POM URL: {}", deploymentSnapshot.getUrl());
+        if (!Objects.equals(deploymentSnapshot.getUrl(), deploymentSnapshotUrl)) {
+            logger.info("  Effective URL: {}", deploymentSnapshotUrl);
+            if (deploymentSnapshotUrl.startsWith("njord:")) {
+                ArtifactStoreTemplate template =
+                        ns.selectSessionArtifactStoreTemplate(deploymentSnapshotUrl.substring("njord:".length()));
+                printTemplate(template, false);
+            }
+        }
+
+        logger.info("");
+
+        List<String> storeNameCandidates = getArtifactStoreNameCandidates(ns);
+        if (storeNameCandidates.isEmpty()) {
+            logger.info("No candidate artifact stores found");
+        } else {
+            logger.info("Locally staged stores:");
+            for (String storeName : storeNameCandidates) {
+                Optional<ArtifactStore> aso = ns.artifactStoreManager().selectArtifactStore(storeName);
+                if (aso.isPresent()) {
+                    try (ArtifactStore artifactStore = aso.orElseThrow()) {
+                        logger.info("  {}", artifactStore);
```

Comment:
which one would be picked by default if there is multiple candidates?

Reply:
Latest (newest):
https://github.com/maveniverse/njord/blob/collapse-config/plugin/src/main/java/eu/maveniverse/maven/njord/plugin3/PublisherSupportMojo.java#L53-L59

Reply:
This one should be highlighted in the emitted list!